assimilation_code/modules/observations/oned_quantities_mod.f90

```diff
@@ -14,13 +14,8 @@
  
 ! BEGIN DART PREPROCESS QUANTITY DEFINITIONS
 !
-!  ! QTY_STATE_VARIABLE is also predefined in the preprocess program.
-!  ! it is being repeated here because for some classes of models
-!  ! this is the only quantity that they will use.  it must be the
-!  ! the first entry in this file and must not be altered.
+!  ! QTY_STATE_VARIABLE is predefined in the preprocess program.
```

Comment:
for all the other quantity files, removing QTY_STATE_VARIABLE seems right.

however, in 1D models this is probably the only QTY a model_mod is going to use.  i'm ok with removing it as long as the comment is a bit more prominent when someone looks at this file.

i wanted to try that suggest thing but i can't see where helen clicked before.  

maybe something like:

! ! QTY_STATE_VARIABLE is a generic quantity which is predefined
! ! by the system and available for code to use without needing
! ! a separate definition here..  It is frequently used in models 
! ! where the items in the state vector do not directly represent 
! ! a physical quantity in the model.    
! !
! ! These additional QTYs are available for more specialized cases.
! !